src/app/project/project.component.ts

```diff
@@ -269,4 +268,9 @@ export class ProjectComponent implements OnInit {
       return 0;
     }
   }
+
+  getProjectGeoAccession() {
+    const content: { geo_series_accessions?: string[] } = this.project?.content;
```

Comment:
not quite understanding this syntax here. if project is null, content can't be the type you're implicitly declaring it to be.

Reply:
It's okay. If project is null, content will be undefined. And using the JS' optional chaining allows us to do this safely https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining